go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cirruslabs/cirrus-ci-annotations
 go 1.12
 
 require (
+	github.com/cirruslabs/go-junit v0.0.0-20220806050453-2f4a6f7dd101
 	github.com/go-test/deep v1.0.1
-	github.com/joshdk/go-junit v0.0.0-20200312181801-e5d93c0f31a8
```

Comment:
Consider using a [`replace` directive](https://go.dev/doc/modules/gomod-ref#replace) insetad:

```
replace github.com/joshdk/go-junit => github.com/cirruslabs/go-junit v0.0.0-20220806050453-2f4a6f7dd101
```

This way only one line needs to be changed now and once the upstream merges the changes.

Reply:
Good point. PTAL [a2be3f1](https://github.com/cirruslabs/cirrus-ci-annotations/pull/41/commits/a2be3f1e9f6a3ade124001ec9a52f01909b80239)